students/mark/Session05/mailroom.py

```diff
@@ -0,0 +1,267 @@
+Jeff Bezos                 $     877.33           1  $      877.33
+Paul Allen                 $     708.42           3  $      236.14
+*/
+
+/*
+
+
+*/
+"""
+
+# donor_db = {"William Gates, III":[326888.82, 326895.67],
+#             "Mark Zuckerberg":[5565.37, 5465.37, 5365.36],
+#             "Jeff Bezos":[877.33],
+#             "paul allen":[236.14, 236.14, 236.14],
+#             "jose gonzalez" : [123.45, 678.90, 101.11]
+#             }
+
+donor_db = {"william gates, iii":[326888.82, 326895.67],
+            "mark zuckerberg":[5565.37, 5465.37, 5365.36],
+            "jeff bezos":[877.33],
+            "paul allen":[236.14, 236.14, 236.14],
+            "jose gonzalez" : [123.45, 678.90, 101.11]
+            }
+
+
+def format_donor_name(donor_names):
+    if ',' in donor_names:
+        name_title=donor_names.split(',')
+        i_var=name_title[0].title() + ','
+        donor_names=i_var.replace(',', "," + name_title[1].upper())
+        return(donor_names)
+    else:
+        name_title=donor_names
+        return(donor_names.title())
+
+
+def print_donor_name(donor_db):
+    """
+    Output the donor name(s)
+    :param: donor_db dict
+    :returns: zero for success
+    """
+    print("Donor list: \n")
+    for donor_names in donor_db.keys():
+        if ',' in donor_names:
+            name_title=donor_names.split(',')
+            i_var=name_title[0].title() + ','
+            donor_names=i_var.replace(',', "," + name_title[1].upper())
+            print(donor_names)
+        else:
+            name_title=donor_names
+            print(donor_names.title())
+    return 0
+
+
+def get_donor(name):
+    """
+    get a donor out of the donor "structure"
+    :params: name/
+    :return: donor
+    """
+    for donor in donor_db:
+        if name.strip().lower() == donor[0].lower():
+            """ get the name in lowercase, makes matching names easier later"""
+            return donor
+
+    return None # or should this be donor?
+
+
+def send_thank_you():
+    """
+    Record a donation, create a thank you letter.
+    """
+
+    # use a while loop to get the user input and execute a function
+    # basic input checking, strip whitespace
+    while True:
+        name = input("Enter a donor's name "
+                     "(or 'list' to see all donors or 'menu' to exit)> ").strip()
+        if name == "list":
+            print_donor_name(donor_db)
+        elif name == "menu":
+            return
+        else:
+            if name.lower() in donor_db.keys():
+                print("name: ", format_donor_name(name), "found.")
+                print(create_letter(name, donor_db))
+            else:
+                if name.lower() not in donor_db.keys():
+                    print("name: ", format_donor_name(name), "is NOT found.")
+                    print("debug: adding a function call to add donor.")
+                    add_donor_info(name, donor_db)
+            break
+
```

Comment:
There is an 'elif' in python, so you can do:
```
if name.lower() in donor_db.keys():
    print(stuff)
elif name.lower() not in donor_db.keys():
    print(different stuff)
break
```
Also, it looks like you only ask how much they are donating if they are a new donor.